resources/bin/build.sh

```
@@ -0,0 +1,89 @@
+
+# ビルドディレクトリを作成
+mkdir -p "$BUILD_DIR"
+
+# ビルド方法を選択（デフォルトは SPM）
+BUILD_METHOD=${1:-spm}
+
+if [ "$BUILD_METHOD" = "direct" ]; then
+    # 方法1: Command Line Tools を直接使用
+    echo "Command Line Tools を使用して直接ビルドしています..."
+    swiftc -o "$BUILD_DIR/$PRODUCT_NAME" CalendarAccessTool.swift \
+        -sdk $(xcrun --show-sdk-path) \
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)